enterprise/admin/actions.py

```diff
@@ -74,7 +74,7 @@ def refresh_catalog(self, request, queryset):  # pylint: disable=unused-argument
 
     # display catalog and task ids that were successfully started refreshing
     updated_message = ''
-    if refreshed_catalogs.items():
+    if refreshed_catalogs:
```

Comment:
Arent these things slightly different logically? I think your change is more correct, b/c otherwise line 77 could throw an Attribute error, but did the linter catch this?

Reply:
Sorry I completely missed this.
Yes this was a complaint from the linter. It falls under the rule of using dictionary iterators without actually iterating.

In some of the instances, I just turned that rule off, but here since we're just checking for existence, items cannot exist without the base and the base doesn't validate as true without items, so it is equitable.